scripts/services/CreateConflationTypeGroups.py

```diff
@@ -81,7 +81,60 @@ def createUiJSON(groups, options):
                         del memberConfig['key']
                         templateDefault = re.findall(r'(?<=\${).*(?=})', memberConfig['default'])
-                        if len(templateDefault) == 1:
+                        # If default points to file of values
+                        if 'HOOT_HOME' in memberConfig['default']:
+                            filePath = os.path.expandvars(memberConfig['default'])
+                            if os.path.isfile(filePath):
+                                with open(filePath, 'r') as jsonFile:
+                                    data = json.load(jsonFile)
+                                    for (key, value) in data.items():
+                                        #Use key as label for what the objects inside represent
+                                        category = key.replace('_', ' ').capitalize()
+
+                                        # ignore comments in the json
+                                        if key != '#' and isinstance(value, list):
+                                            for categoryObj in value:
+                                                memberObject = {
+                                                    'id': match,
+                                                    'input': 'text',
+                                                    'type': 'json',
+                                                    'parentKey': key
+                                                }
+                                                objKeys = []
+
+                                                if 'key' in categoryObj:
+                                                    categoryKey = categoryObj['key']
+                                                    objKeys.append('key')
+                                                    memberObject['key'] = categoryKey
+                                                else:
+                                                    categoryKey = ''
+
+                                                if 'value' in categoryObj:
+                                                    categoryVal = categoryObj['value']
+                                                    objKeys.append('value')
+                                                    memberObject['value'] = categoryVal
+                                                else:
+                                                    categoryVal = '*'
+                                               
+                                                if 'distance' in categoryObj:
+                                                    categoryDefault = categoryObj['distance']
+                                                    objKeys.append('distance')
+                                                    memberObject['distance'] = categoryDefault
+                                                else:
+                                                    categoryDefault = ''
```

Comment:
This is the only non-generic bit right?
We'd need to refactor PoiSearchRadii.json to avoid this like so:
From
```
    {
      "key": "amenity",
      "value": "grave_yard",
      "distance": "1000"
    },
```
To
```:
    {
      "matcher": {
        "key": "amenity",
        "value": "grave_yard"
      },
      "default": {
        "key": "distance",
        "value": "1000"
      }
    }
```

But I'm still not sure this format is robust enough for other possible json configs.

Reply:
yea but I feel like the key value portion could change meaning cause maybe value for a different json object would be the input default.
At 1 point i wondered if distance should be switched to value and value should be something like `keySpecifier`

```
{
  "key": "amenity",
  "keySpecifier": "grave_yard",
  "value": "1000"
}
```

Reply:
When we say "other possible json configs", what are we thinking about? Other conflation algs allowing per type distances or something else?

Reply:
any other hoot param that accepts a json a json as it's value that we override from the ui

Reply:
Ok, I see. I think this is the first example of that we have. The only other I can think of right now is the conflate tag filtering, but that hasn't been integrated with the UI yet.

Reply:
Ok, that tag filtering looks like there wouldn't be a list of defaults, but rather a DSL to build the filter expression, so that would be different (and another one-off) from the config driven advanced options UI.

I think we'll merge this as-is and if another config comes up that we want to expose we can design a generic pattern for the config json then.  One example might be `poiPolygonPoiTagIgnoreList` which isn't json at the moment but might be made to be if we needed to add or remove some of those from a conflate job.